packages/protocol/contracts-0.8/common/EpochManager.sol

```diff
@@ -51,13 +51,18 @@ contract EpochManager is
 
   uint256 public firstKnownEpoch;
   uint256 internal currentEpochNumber;
+  address[] internal electedAccounts;
```

Comment:
Since this contract is already deployed on alfajores we should try not to break storage

Reply:
made public